### ISSUE TITLE
context.document.body.getHtml() fails on chrome and edge

### ISSUE THREAD

**Issue by nickolasfisher** [OfficeDev/office-js#2933]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

# Provide required information needed to triage your issue

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Office on the web
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: ______
* Operating System: Windows 10
* Browser (if using Office on the web): Chrome

## Expected behavior

Calling `context.document.body.getHtml()` should result in the html of the word document being loaded.

## Current behavior

Expected behavior works in both Firefox and desktop versions of Word however Chrome is returning the error

```
WordEditorDS.box4.dll3.js:613 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'dispose')
    at mk.KG (WordEditorDS.box4.dll3.js:613:135)
    at WordEditorDS.box4.dll3.js:1502:365
```

## Steps to reproduce

I have put up a minimal app [on github here](https://github.com/nickolasfisher/WordAddinFailing)

1. Start the application with `npm start:web`
2. Open the document `example03.docx` in the `sampledocs` directory of the application
3. Run the add in in the context of `example03.docx`
4. Click the **Run** button
5. Note the error in the console

## Link to live example(s)
Github repo linked above

# Provide additional details
N/A

## Context

My add in leverages the Semantic elements from the HTML tags to properly analyze the document.  I lose that value if I load the text and loading OOXML would require a complete overhaul this analysis.

## Useful logs

- [ X ] Console errors
- [ ] Screenshots
- [ X ] Test file (if only happens on a particular file)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by ghost** [comment]
Thank you for letting us know about this issue. We will take a look shortly. Thanks.

**Comment by RuizhiSunMS** [comment]
@nickolasfisher . We can't reproduce the issue, could you please confirm the issue is still blocking you?

**Comment by nickolasfisher** [comment]
![image](https://user-images.githubusercontent.com/56088064/200327124-0d8be5ef-3194-464e-9c7d-c4eaeea9943c.png)

@RuizhiSunMS  I can confirm the issue still exists on Chrome though I cannot repro on Edge anymore

```
Chrome is up to date
Version 107.0.5304.88 (Official Build) (64-bit)
```

```
OS Name	Microsoft Windows 10 Pro
Version	10.0.19043 Build 19043
```

**Comment by RuizhiSunMS** [comment]
Something very strange. Our colleague just can't reproduce the issue on any browser.
How is the status if you clear the cache?
Or can you provide a screen recording?

**Comment by nickolasfisher** [comment]
@RuizhiSunMS Yes, recording attached.

The recording below was taken on a 3rd machine (the first machine is my primary development machine, the second was a collegues, this third is an AWS Work Space instance).

For reference I've attempted to this on `node version 18.11.0` with `npm version 8.19.2` and `node version 16.13.2` with `npm version 8.1.2`  I deleted `node_modules` and reran `npm i` after switching versions each time.

Clearing the cache produces the same error.

https://user-images.githubusercontent.com/56088064/200604049-4e9046d1-cb34-400d-b05f-f65dd744be5b.mp4

**Comment by RuizhiSunMS** [comment]
@nickolasfisher 
Thanks for the attachment. 
However, unfortunately we still can't reproduce the issue. Maybe answer the below questions could help.
1. the same file works fine on Edge but is caught by the problem on the Chrome?
2. does every file have the problem or only some specific files? 
3. can you share your file with us?

**Comment by nickolasfisher** [comment]
Actually today its failing on Edge.  Recording below.  I checked both my AWS machine and my primary development machine.    

https://user-images.githubusercontent.com/56088064/200844352-22689829-7163-420b-bb8e-ddcd597391b0.mp4

The problem exists in all files as far as I can tell.  See recording below for the issue occurring on a new document.  I can provide more examples if needed but doesn't seem neccessary.  

https://user-images.githubusercontent.com/56088064/200844969-97d4606f-9189-425c-b446-38491eafb413.mp4

The file I'm using was included in the Github repo I supplied at the beginning of this ticket.  Please let me know if you're having trouble accessing it.

Just for reference, below is the same functionality **succeeding** in Firefox

https://user-images.githubusercontent.com/56088064/200846020-0ffdc7df-56f5-4832-9ef7-2b74d257598f.mp4

**Comment by xiruatms** [comment]
hi,  @nickolasfisher , thanks for reporting this with all these useful information. We're working on this now.

As a mitigation, will you please take a try switching to "Web View" if possible? At the bottom right of the browser, you might notice there will be "Page View" and "Web View" in some cases. If there're, please make sure "Web View" is selected and run your test agin. 

![image](https://user-images.githubusercontent.com/40648203/200849294-2443728d-b914-40ca-bdd1-b75acc7d06e6.png)

Please let us know whether this works.

**Comment by nickolasfisher** [comment]
@xiruatms  we're on to something.  Web View does indeed work

Please see attached video

https://user-images.githubusercontent.com/56088064/200851013-bb3567ca-cc25-414f-8c36-02feebd4735c.mp4

**Comment by xiruatms** [comment]
@nickolasfisher , thanks for your confirmation. Hope this can unblock your work to some extent. We're working on confirming a better solution to handle the case you reported. The fix might reach production soon.

**Comment by nickolasfisher** [comment]
@xiruatms verifying with a colleague that this resolves his problem as well but yes, this should temporarily unblock us.  If it causes further problems I'll let you know.  Can I be alerted as to when this reaches production as this does create a work around for our clients.

Thank you for your help

**Comment by xiruatms** [comment]
@nickolasfisher , surely I will keep you posted if the fix is confirmed on production. Thanks a lot for reporting this to us.

**Comment by RuizhiSunMS** [comment]
@nickolasfisher Does this issue still block you? One guy from the owning team said they had a release.

**Comment by nickolasfisher** [comment]
@RuizhiSunMS  sorry for the delay, we are still experiencing the issue.

**Comment by nickolasfisher** [comment]
@RuizhiSunMS any update on this?  We are still experiencing the issue
